.github/workflows/run-forge.yaml

```diff
@@ -65,6 +65,13 @@ jobs:
           ref: ${{ inputs.GIT_SHA }}
           # get the last 10 commits if IMAGE_TAG is not specified
           fetch-depth: env.IMAGE_TAG != null && 0 || 10
+      - uses: actions/setup-python@v4
+        if: env.FORGE_ENABLED == 'true'
+        with:
+          python-version: '3.9'
```

Comment:
Let's not do this, this step reads from `.python_version` (which is also 3.9). Good to keep a uniform version.